test/util/TestData.scala

```diff
@@ -110,155 +109,192 @@ trait TestData extends MockitoSugar {
   val dan                           = "123456"
   val bacs                          = "BACS"
 
-  lazy val dutyDefermentStatementFiles: Seq[DutyDefermentStatementFile] = List(
-    DutyDefermentStatementFile(
-      "someFilename",
-      "downloadUrl",
-      fileSizeData,
-      DutyDefermentStatementFileMetadata(
-        previousMonthDate.getYear,
-        previousMonthDate.getMonthValue,
-        periodStartDay,
-        previousMonthDate.getYear,
-        previousMonthDate.getMonthValue,
-        periodEndDay,
-        FileFormat.Csv,
-        DutyDefermentStatement,
-        Weekly,
-        Some(true),
-        Some(bacs),
-        dan,
-        None
-      )
-    ),
-    DutyDefermentStatementFile(
-      "someFilename2",
-      "downloadUrl",
-      fileSizeData,
-      DutyDefermentStatementFileMetadata(
-        previousMonthDate.getYear,
-        previousMonthDate.getMonthValue,
-        periodStartDay,
-        previousMonthDate.getYear,
-        previousMonthDate.getMonthValue,
-        periodEndDay,
-        FileFormat.Pdf,
-        DutyDefermentStatement,
-        Supplementary,
-        Some(true),
-        Some(bacs),
-        dan,
-        None
-      )
-    ),
-    DutyDefermentStatementFile(
-      "someFilename3",
-      "downloadUrl",
-      fileSizeData,
-      DutyDefermentStatementFileMetadata(
-        previousMonthDate.getYear,
-        previousMonthDate.getMonthValue,
-        periodStartDay,
-        previousMonthDate.getYear,
-        previousMonthDate.getMonthValue,
-        periodEndDay,
-        FileFormat.Csv,
-        DutyDefermentStatement,
-        Excise,
-        Some(false),
-        Some(bacs),
-        dan,
-        None
-      )
-    ),
-    DutyDefermentStatementFile(
-      "someFilename4",
-      "downloadUrl",
-      fileSizeData,
-      DutyDefermentStatementFileMetadata(
-        twoMonthsPriorDate.getYear,
-        twoMonthsPriorDate.getMonthValue,
-        periodStartDay,
-        twoMonthsPriorDate.getYear,
-        twoMonthsPriorDate.getMonthValue,
-        periodEndDay,
-        FileFormat.Pdf,
-        DutyDefermentStatement,
-        Excise,
-        Some(false),
-        Some(bacs),
-        dan,
-        None
-      )
+  def getDdSttMetadata(
```

Comment:
Better to rename it to either ddSttMetadata or createDdSttMetadata as get is java convention not scala, and here we are just populating the DutyDefermentStatementFileMetadata